### DISCUSSION TITLE
Why there is a div element without a CSS class inside .mantine-RadioGroup-root

### DISCUSSION THREAD

**Discussion by mohammad-reza-rahbar** [mantinedev/mantine#6624] [Q&A]
I am trying to implement a segmented control similar to the [SegmentedControl](https://mantine.dev/core/segmented-control/) using the [Radio.Card component](https://mantine.dev/core/radio/#radiocard-component) with Radio.Group and want to make the height of it flexible. But I have to select a `div` inside `.mantine-RadioGroup-root` using the child combinator `>`  to achieve that. Am I doing the write thing or there is a better way to make the height of Radio.Group and all its Radio.Cards to fill the height of its parent?

**Comment by rtivital** [comment]
What's wrong with using `>`?

**Reply by mohammad-reza-rahbar** [reply]
I don't know. Having an element with class name feels more robust than an element without a class name, in the sense that,
it is more probable to be changed in a future update and breaks my code. By the way, thanks for such a nice library. I am really looking forward to the next minor update.
